target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -84,6 +84,10 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:4" "3:lan:3" "4:lan:2"
 		;;
+	belkin,f9j1108-v2)
+		ucidef_add_switch "switch0" \
+			"0@eth1" "2:lan" "3:lan" "4:lan" "5:lan" "6@eth0" "1:wan"
```

Comment:
two CPU ports on the same switch?

Reply:
Since this device is pretty much the same hardware as the archer-c7, then I set up the switch in the same way.
How can I verify that it is correctly configured ?

Reply:
Then instead of creating a new case that is exactly the same...
add it to the case including archer-c7 at line 294

Reply:
Good point :)